src/NServiceBus.Core/DelayedDelivery/TimeoutManager/TimeoutManager.cs

```diff
@@ -69,11 +55,95 @@ protected internal override void Setup(FeatureConfigurationContext context)
 
                 return new ExpiredTimeoutsPoller(b.Build<IQueryTimeouts>(), b.Build<IDispatchMessages>(), dispatcherAddress, circuitBreaker);
             }, DependencyLifecycle.SingleInstance);
+
+            context.RegisterStartupTask(b => new TimeoutPollerRunner(b.Build<ExpiredTimeoutsPoller>()));
+        }
+
+        static string SetupDispatcherSatellite(FeatureConfigurationContext context, RetryStorageCleaner storageCleaner)
+        {
+            var errorQueueAddress = ErrorQueueSettings.GetConfiguredErrorQueue(context.Settings);
```

Comment:
it seems that the fix on v5 also added some acceptance test: https://github.com/Particular/NServiceBus/commit/ed0e2c96ec1f4a7b7eab8367f0140f3760174b77

so I'd say separate PR